pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -189,8 +193,8 @@ def check_valid(self):
             # 3. decode() yields None
             first_frame = next(cont.decode(video=0), None)
             if first_frame is None:
-                raise av.AVError("Video does not contain any frames")
-        except av.AVError:
+                raise VideoDoesNotContainFramesError()
+        except (av.AVError, VideoDoesNotContainFramesError):
             return False
```

Comment:
Do we use the error somewhere else? Because we immediately catch it, I thought we could just:
```python
if first_frame is None:
    # No frames
    return False
````